plugins/git/service/src/gitservice.cpp

```diff
@@ -60,7 +60,53 @@ int gitDiffToStringCompactCallback(
   return 0;
 }
 
+#define GIT_MACRO_TO_THRIFT_TYPE(GIT_TYPE, THRIFT_TYPE) \
+  case GIT_TYPE: \
+    return cc::service::git::GitObjectType::THRIFT_TYPE;
+
+#if LIBGIT2_VER_MAJOR == 1 || LIBGIT2_VER_MINOR >= 28
```

Comment:
Wouldn't we like to get prepared for the future by saying `LIBGIT2_VER_MAJOR >= 1`?